lib/ace/ext-beautify.js

```diff
@@ -0,0 +1,334 @@
+ace.define("ace/ext/beautify/php_rules",["require","exports","module","ace/token_iterator"], function(require, exports, module) {
```

Comment:
Are you actually using all these extensions? I would prefer to see just the ace editor minified file and maybe a couple config files, not 27 new files.